### DISCUSSION TITLE
Link allows setting "Open with new Tab"

### DISCUSSION THREAD

**Discussion by listill** [outline/outline#2142] [Ideas]
Can you consider adding an option of "Open with new Tab" to the Link?

**Comment by tommoor** [comment]
Hi, not really – as it's standard browser behavior to open a new tab when you `cmd`+click on any webpage link, did you mean something else?

**Reply by listill** [reply]
Yes, I know cmd+click, but when I try to set the link as a shared document link, clicking the link does not automatically refresh the page.

Sorry, my English is a little poor. I hope you can understand it.

Thanks.

**Reply by tommoor** [reply]
Yes, sorry it's a little difficult to understand – are you able to upload a video of the behavior?

**Reply by listill** [reply]
https://user-images.githubusercontent.com/15646491/118406161-a9d77800-b6ad-11eb-9549-d3c50413c01f.mp4

When I click on a link, it will not jump to a new page.

**Reply by tommoor** [reply]
This seems like a bug, it might be specific to internal links on shared pages – I'll convert this to a bug.
